### PR TITLE
LRDOCS-8202 Collection Pages Review

### PR DESCRIPTION
**Pull request by jorge-campo**
*This is the draft for the Collection Pages content described in LPS-111154.*

New articles in Content Authoring and Management section:

- about-collections-and-collection-pages.md
- creating-collections.md
- displaying-collection-and-collection-pages.md

New article in Experience Personalization section:

- personalizing-collections.md (updates and replaces content-set-personalization.md)
[LRDOCS-8202](https://app.gitkraken.com/glo/card/893b45545dd9425ba9e82444046d4aab)

### CODE REVIEW

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+A Collection Display Fragment is a type of Fragment that shows a Collection. You can use this Fragment to show your Collection on any Content Page.
+
+![You can show the Collection's content using a Collection Display Fragment](./about-collections-and-collection-pages/images/02.png)
+
+For more information, see [Displaying Collection](./displaying-collections.md).
+
+## Customizing Collections
+
+You can associate a Liferay Segment to your Collection to display different Collection's content based on the [Segment](../../site-building/personalizing-site-experience/segmentation/creating-and-managing-user-segments.md).
```
Comment (JorgeFerrer) [inline docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:37]: I had to reread this sentence a few times to understand it. 
I wonder if "Collection items" is easier to read and understand than "Collection's content"

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    For example, for your online Kitchenware Store, you create a Collection including all blog posts with the *porcelain* tag (this tag represents your criteria.) The Dynamic Collection automatically includes all the new blog post with the *porcelain* tag.
+
+For information about how to create Manual or Dynamic Collections, see [Creating Collections](./creating-collections.md).
+
+## Displaying Collections
+
+You can display collection using a [Collection Page](./displaying-collections-and-collection-pages.md#displaying-collections-using-a-collection-page) or a [Collection Display Fragment](./displaying-collections-and-collection-pages.md#displaying-collections-using-a-collection-display-fragment).
+
+A Collection Page is a type of page where the content is linked to a Collection. Collection Pages ease the task of showing and customizing Collections.
```
Comment (JorgeFerrer) [inline docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:25]: I think that it's more accurate to say that that the page is linked to the Collection, not its content. By being linked to the collection, the editor of the page makes it easier to display the items in the collection and (in the near feature) add controls and filters related to the collection.

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+By default, any user can see all the Collection's content. However, you can create *Personalized Variations* of these items using Segments. For more information, see [Customizing Collections Using Segments](./customizing-collections-using-segments.md).
+
+```note::
+   Before displaying a Collection, you must create one. For more information, see [Creating Collections](./creating-collections.md).
+```
+
+## Displaying Collections Using a Collection Page
+
+A Collection Page is a type of page where the content is linked to a Collection.
```
Comment (JorgeFerrer) [inline docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md:13]: Same as above, it's more accurate to say that

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+![You can display your Collection using a Collection Page](./about-collections-and-collection-pages/images/01.png)
+
+A Collection Display Fragment is a type of Fragment that shows a Collection. You can use this Fragment to show your Collection on any Content Page.
```
Comment (jkappler) [inline docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:29]: Here we could also mention that this is the same for page templates and display pages

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
```
Comment (JorgeFerrer) [inline docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md:14]: One question that we are starting to receive and we'll most probably continue to receive is "What's the difference between a Collection Page and a regular page which a Collection Display fragment or Asset Publisher configured to display the same collection?"

So I wonder if we should add a note or hint here and answer the question proactively. What do you guys think?

As of 7.3 the answer would be that creating pages that display collections is ver common in sites and Collection Pages attempt to make the creating process easier and also simplify the management, for example by providing a quick link in the top bar to create new items. We could also add that in the future we will keep adding more features to facilitate the most common tasks that site admins perform for collection pages.

Thoughts?

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md (hunk 1 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:37]: Sentence was rewritten.

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md (hunk 5 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md:14]: This is a question that I discussed with @jkappler as well. I will highlight the fact that Collection Pages ease Collection management. I'm aware that Collection Pages will include new functionalities soon, and we'll update the documentation accordingly to reflect this fact.

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md (hunk 2 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:25]: The sentence was rewritten.

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md (hunk 3 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/displaying-collections-and-collection-pages.md:13]: The sentence was rewritten.

File: docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md (hunk 4 above)
Comment (jorge-campo) [reply docs/dxp/7.x/en/content-authoring-and-management/collections-and-collection-pages/about-collections-and-collection-pages.md:29]: The information was included.

**Comment by jorge-campo** [comment]
Superseded by https://github.com/sez11a/liferay-learn/pull/174

**Pull request closed without merging**